src/components/do-dont-list/DoDontList.tsx

```diff
@@ -49,7 +49,14 @@ const DoDontItem: React.FC<DoDontItemProps> = ({ children, listItemType, ...rest
   const listItem = useContext(DoDontListContext);
   return (
     <li {...rest}>
-      {(listItemType || listItem) === 'do' ? <Tick /> : <Cross />}
+      {(listItemType || listItem) === 'do' ? (
+        <Tick />
+      ) : (
+        <>
+          <Cross />
+          do not{' '}
+        </>
```

Comment:
This ought to be a prop with a default value, so that people can disable it if required. I know there are places that the DoDontList is used where they wouldn't want this text, or be able to change/disable it. Maybe `prefixText`?